config/routes.rb

```diff
@@ -31,7 +31,11 @@
   resources :custom_manifests, only: [:create, :update, :destroy, :show]
 
   scope "user" do
-    resources :counters, controller: "user_counters", only: [:index, :update]
+    resources :counters, controller: "user_counters", only: [:update] # Legacy route for GodTools Android v5.7.0-v6.0.0
```

Comment:
I'm hoping we will only need to keep this legacy route for 6-12 months to give most users a chance to upgrade. Then we can delete this legacy route